README.md

```diff
@@ -15,7 +15,7 @@
 
 # Introduction
 
-Design patterns are the best formalized practices a programmer can use to
+Design patterns are the best-formalized practices a programmer can use to
```

Comment:
This seems to be okay and need not be improved.

Reply:
Sorry for that mistake from my part, I have corrected.